src/components/Rankings.tsx

```diff
@@ -41,7 +42,7 @@ export default async function Rankings({
         );
 
         if (zone == null) {
-            throw new Error(`Zone with encounter ${encounter} not found`);
+            notFound();
```

Comment:
Why is not all cases handled here? Why are some in the page.tsx instead?

Reply:
But shouldn't "page.tsx" also check for zone? if page.tsx does a process of checking the parameters, why does it not check zone parameter as well?